### ISSUE TITLE
Login.js Ln 58

### ISSUE THREAD

**Issue by walshe** [AnomalyInnovations/serverless-stack-demo-client#23]
how come this line is missing 0 as mentioned in the tutorial

this.props.history.push("/");

**Comment by jayair** [comment]
@walshe Which line are you talking about in which part of the tutorial? Line 58 in Login.js is this one https://github.com/AnomalyInnovations/serverless-stack-demo-client/blob/master/src/containers/Login.js#L58.

**Comment by walshe** [comment]
yes line 58 in Login.js .. where has the this.props.history.push("/"); right after the user has been marked as authenticated?

**Comment by jayair** [comment]
@walshe I'm sorry I don't understand what you are saying. Can you elaborate?

**Comment by walshe** [comment]
what I am trying to say is this:

this line is missing in Login.js:
this.props.history.push("/")

**Comment by jayair** [comment]
@walshe No we don't need that line after we secure our pages. It's in this chapter - https://serverless-stack.com/chapters/redirect-on-login.html

**Comment by walshe** [comment]
ah I see :)

Btw, another question.. doesnt Cognito have some way of easily integrating default (but customizable) signup/login forms that be used in a web app (similar to auth0) so that a developer does not have to go to the trouble ?  and if so would it be nice to add to the tutorial @jayair
